test/integration/targets/vmware_host_hyperthreading/tasks/main.yml

```diff
@@ -0,0 +1,83 @@
+# Test code for the vmware_host_acceptance module.
+# Copyright: (c) 2018, Abhijeet Kasurde <akasurde@redhat.com>
```

Comment:
Add your name here.

Reply:
Sorry, I copied the tests from one of your modules.